Sources/EntitiesMP/Common/Particles.cpp

```diff
@@ -1591,7 +1591,7 @@ INDEX Particles_Regeneration(CEntity *pen, FLOAT tmStart, FLOAT tmStop, FLOAT fY
     
     UBYTE ubR = (UBYTE) (192+afStarsPositions[iRnd][1]*64);
     UBYTE ubG = (UBYTE) (192+afStarsPositions[iRnd][2]*64);
-    UBYTE ubB = (UBYTE) (192+afStarsPositions[iRnd][3]*64);
+    UBYTE ubB = (UBYTE) (192+afStarsPositions[iRnd][3]*64); // FIXME DG: afStarsPositions[i][3] invalid, what's going on here?
```

Comment:
I saw Clang fire this warning, too...I think this is a bug in the original code, and these should have counted from 0 to 2, not 1 to 3, but that's just a guess.